### PR TITLE
Chore: Add missing logos, convert to svg, add Ajv

### PR DESCRIPTION
**Pull request by brianwarner**
A number of project logos were missing, they have been added. In addition, the project logos were linked to a png, which was pixellating at small size. The .svg was available in the artwork repo, so the links have been updated. Finally, Ajv had not been added to the list as an incubating project.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,63 +16,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
-| | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/master/CONTRIBUTING.md) |  |
+| <img alt="logo of project ajv" src="https://github.com/openjs-foundation/artwork/raw/main/projects/ajv/ajv-logo-color.svg" width="30" />  | [Ajv](https://ajv.js.org/) |   | [Contributing Guide](https://ajv.js.org/contribute.html) | [Code of Conduct](https://github.com/ajv-validator/ajv/blob/master/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project NativeScript" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nativescript/nativescript-icon-color.svg" width="30" /> | [NativeScript](https://nativescript.org/) |   | [Contributing Guide](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md) | [Code of Conduct](https://github.com/NativeScript/codeofconduct) |
+| <img alt="logo of project nvm" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nvm/nvm-logo-grayscale.svg" width="30" />  | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/master/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:76]: lol i guess this is nvm's logo now?
```suggestion
| <img alt="logo of project nvm" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nvm/nvm-logo-grayscale.svg" width="30" />  | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/HEAD/CONTRIBUTING.md) |  |
```

Comment (brianwarner) [reply README.md:76]: It's the one you and I discussed a while back, yeah :-)

Comment (brianwarner) [reply README.md:76]: If there's another one to use, we can definitely sub it out. Just seemed better than leaving it empty...

Comment (ljharb) [reply README.md:76]: nah this is fine, i just didn't remember where it came from

Comment (brianwarner) [reply README.md:76]: Alright cool. I guess it was a placeholder that stayed a placeholder. Easy enough to swap at any point though.

**Review by joesepi — APPROVED** [review]
LGTM. Thank you, @brianwarner

**Review by ljharb — APPROVED** [review]

**Pull request merged**